yt/frontends/gadget/tests/test_outputs.py

```diff
@@ -147,3 +154,48 @@ def test_magneticum():
     for test in sph_answer(ds, "snap_132", 3718111, mag_fields, center="max"):
         test_magneticum.__name__ = test.description
         yield test
+
+
+camels_kwargs = {
+    "bounding_box": [[8126.0, 22126.0], [5140.0, 19140.0], [5500.0, 19500.0]]
+}
+
+
+@requires_module("h5py")
+@requires_file(magneticum_camels)
+def test_magneticum_camels():
+    # In this test, we're only checking the metal fields since this
+    # is a dataset with special metal handling
+    ds = data_dir_load(magneticum_camels, kwargs=camels_kwargs)
+    dd = ds.all_data()
+    elems = [
+        "He",
+        "C",
+        "Ca",
+        "O",
+        "N",
+        "Ne",
+        "Mg",
+        "S",
+        "Si",
+        "Fe",
+        "Na",
+        "Al",
+        "Ar",
+        "Ni",
```

Comment:
I noticed your `MetalMasses_` definition didn't include a definition for `14`. Did you want to add that too since that's the use case for your test?

Reply:
So `14` has to be treated differently. It's not a real element, it's basically just the mass of the rest of the elements not specified in the list, and in the codes that use it it's called `Ej` for "ejecta" (because it comes from supernovae explosions). We define an `Ej_fraction` field, but the reason I can't stick it in the `elem` list is that list the `Ej_mass` field doesn't exist, as it's not a real element and it doesn't get passed through yt's species machinery (though maybe it should be defined on its own).

Reply:
Ok so your new commits do define it on its own now, is my understanding of that commit correct?